src/com/xilinx/rapidwright/edif/EDIFTools.java

```diff
@@ -755,7 +755,7 @@ public static EDIFNetlist loadEDIFFile(Path fileName) {
     public static EDIFNetlist loadEDIFFile(Path fileName, int maxThreads) {
         try {
             final long size = Files.size(fileName);
-            if (ParallelEDIFParser.calcThreads(size, maxThreads) > 1) {
+            if (ParallelEDIFParser.calcThreads(size, maxThreads, fileName.toString().endsWith(".gz")) > 1) {
```

Comment:
```suggestion
            long estimatedUncompressedSize = size * EDIF_GZIP_RATIO; 
            if (ParallelEDIFParser.calcThreads(estimatedUncompressedSize, maxThreads) > 1) {
```
(After moving the `EDIF_GZIP_RATIO` member over from `ParallelEDIFParser`)
Could this be more meaningful/generic?

Reply:
Not sure that this works in the uncompressed case.

Reply:
Oh yes, sorry! What about gating the inflation using the `.gz` check? My overall suggestion was that we can re-inflate the size back to the uncompressed value before calling the EDIF parser.

Reply:
I don't think I understand the motivation of why this code needs to be modified.  The only thing this check does it decide between parallel parsing and serial parsing of the file based on size.

Reply:
To me, it's a little easier to understand/general if we always provide the uncompressed size to a method than to have something specific happen inside the parser. Scenarios like what if the exact size of the file becomes known, or if we have to support some other compression standard, becomes less intrusive to support. Only a suggestion, not anything to hold up this PR. Let's move on.